pandas/compat/numpy/__init__.py

```diff
@@ -9,6 +9,7 @@
 np_version_under1p21 = _nlv < Version("1.21")
 np_version_under1p22 = _nlv < Version("1.22")
 np_version_gte1p22 = _nlv >= Version("1.22")
+np_version_gte1p24 = _nlv >= Version("1.24")
```

Comment:
Was this added since this PR is to be backported and it's in the 1.5.x branch?

Reply:
I added it in the second commit (the one after the pure revert, which also catches the `SystemError`)

TBH I accidentally left it in after I was trying things out, and then figured I'd just leave it here as it'll probably be useful at some point. Reckon it needs removing?

Reply:
Yeah I would opt to only add this until we really need it (and IIRC you may have wanted to remove these in favor of direct version checks in the code?)

Reply:
that was only for the Python version, because pyupgrade will rewrite them (they don't rewrite based on 3rd party libraries)

OK, sure, I'll remove this line - is the rest alright?

Reply:
Yup the rest LGTM